src/edu/stanford/nlp/tagger/maxent/Dictionary.java

```diff
@@ -38,8 +38,7 @@ public Dictionary() {
 
   void fillWordTagCounts(Map<String, IntCounter<String>> wordTagCounts) {
     for (Map.Entry<String, IntCounter<String>> wordTagCount : wordTagCounts.entrySet()) {
-      TagCount count = new TagCount(wordTagCount.getValue());
-      dict.put(wordTagCount.getKey(), count);
+      dict.put(wordTagCount.getKey(), new TagCount(wordTagCount.getValue()));
```

Comment:
Again, is this an efficiency issue or a compact code issue?

Reply:
Similar to above. The Java compiler does a quite literal translation, so the resulting bytecode with the local variable is usually about 2 bytes more - an astore and an aload to a local register. This will be optimized away by the JIT compiler easily, but may affect the decision when to inline a function (which depends, to my understanding, a lot on bytecode size - functions with fewer than 12 are always inlined unless multimorphic, and when code is considered to be hot also larger functions will be inlined eventually).